packages/p2p/src/constants/business-hour-times.ts

```diff
@@ -0,0 +1,50 @@
+export const hours_list = [
```

Comment:
Do they need translations?

Reply:
i think yes, but i think for now I will leave this here, and add a TODO since its not being displayed yet to users.